### ISSUE TITLE
QGIS 3.14 Start Up Failure After MS WIN 10 Security Update 22AUG2020

### ISSUE THREAD

**Issue by fenem** [qgis/QGIS#38432]
**Describe the bug**
After allowing the WIN10 Update and restarting; QGIS is throwing the following error:

**Could not load qgis_app.dll.  Windows Error: The specified module could not be found.  Check C:\OSGEO4W64\bin\gqis-bin-g7.env for correct environment paths.**

**How to Reproduce**

**QGIS and OS versions**

NOT AVAILABLE SINCE APPLICATION DID NOT START

**Additional context**

- QGIS 3.14 was working just fine prior to the WIN10 update.  
- I have checked the environment paths and all looks good.
- The qgis_app.dll IS INDEED on the system and has not been moved.

**Comment by gioman** [comment]
Cannot replicate on my testing Windows 10 machine.

**Comment by fenem** [comment]
Giovanni,

My problem did not manifest until after the latest WIN10 security update; twp days ago.

Have you installed the WIN 10 update?

thx

Michael Fene’

Michael Fene’
Profit Laser Marketing
Carlsbad, California 92010
________________________________

**Comment by spono** [comment]
same issue here, using 3.10.x . GRASS available but not recognised and all PATHs are ok.
I noticed it only after the last version update (through OSGeo) and now I'm unable to use `qgis-ltr-bin-g7`; `qgis-ltr-bin` works but GRASS modules are not available.

I don't know if somewhow it may help, but I found these resources ([1](https://github.com/conda-forge/qgis-feedstock/issues/43), [2](https://trac.osgeo.org/osgeo4w/ticket/601)) from other forums.

**Comment by spono** [comment]
sorry, I noticed only now the solution on another issue ([#39310](https://github.com/qgis/QGIS/issues/39310)).

@fenem try this and see if it works for you.

**Comment by gioman** [comment]
Not a general issue.

**Comment by FlashThunder** [comment]
I got the same problem with QGiS 3.16
